committers.md

```diff
@@ -42,6 +42,7 @@ navigation:
 |Kazuaki Ishizaki|IBM|
 |Xingbo Jiang|Databricks|
 |Holden Karau|Apple|
+|Attila Zsolt Piros|Cloudera|
```

Comment:
Same comment here. (I agree this is quite confusing.)

Actually there's a convention in the list if I understand correctly (I don't know where this is documented) - the list is alphabetically sorted by "last name" (not first name).

Could you please check the sequence a bit and please update the same if the list follows the pattern?